packages/grid/_modules_/grid/hooks/features/export/seralizers/csvSeraliser.ts

```diff
@@ -1,15 +1,32 @@
-import { GridColumns, GridRowId, GridRowModel } from '../../../../models';
+import { GridCellValue, GridColumns, GridRowId, GridRowModel } from '../../../../models';
 
-export function buildRow(row: GridRowModel, columns: GridColumns): Array<GridRowModel> {
-  const mappedRow: GridRowModel[] = [];
-  columns.forEach((column) => column.field !== '__check__' && mappedRow.push(row[column.field]));
+const serialiseCellValue = (value) => {
+  if (typeof value === 'string') {
+    const formattedValue = value.replace(/"/g, '""');
+    return formattedValue.includes(',') ? `"${formattedValue}"` : formattedValue;
+  }
+  return value;
+};
+
+export function serialiseRow(
+  row: GridRowModel,
+  columns: GridColumns,
+  getCellValue: (id: GridRowId, field: string) => GridCellValue,
+): Array<string> {
+  const mappedRow: string[] = [];
+  columns.forEach(
+    (column) =>
+      column.field !== '__check__' &&
```

Comment:
This one `'__check__'` should be a constant

Reply:
Done